### PR TITLE
Update EFSL for Specifications and Javadoc

### LINKED ISSUE #317: Update EFSL for Specifications and Javadoc
Per the discussions on the Spec Committee and Platform Dev mailing lists, it's been discovered that many of the Javadoc and Spec references to the EFSL need updating. Please reference the following required updates and keep them in mind as your Spec Project is updating the files for Jakarta EE 9.

Note: Some Spec Projects have already started or even completed these updates. If so, just kindly return/close this Issue. Thanks!

**Required EFSL updates for Javadoc**

For javadoc, the EFSL.html located in src/main/javadoc/doc-files should be modified as follows:
* the `<<url to this license>>` needs to be replaced with efsl.php link[1]
* the `[title and URI of the Eclipse Foundation specification document]` needs to be replaced with the Specification Name and URL (Reference [2] for an example.)
* The javadoc footer copyright year needs to be updated to 2018, 2020 as defined in the pom.xml

**Required EFSL updates for Specifications**

For specification, the license-efsl.adoc located in src/main/asciidoc should be modified as follows:
* Update copyright year to 2018, 2020 from 2019. Add link to EFSL.
* the `<<url to this license>>` needs to be replaced with efsl.php link[1]
* the `[title and URI of the Eclipse Foundation specification document]` needs to be replaced with the Specification Name and URL (Reference [2] for an example.)

[1] https://www.eclipse.org/legal/efsl.php  
[2] https://jakarta.ee/specifications/enterprise-beans/4.0/

### PR DESCRIPTION
**Pull request by mcserra**
Fixes #317

### CODE REVIEW

File: api/src/main/javadoc/doc-files/speclicense.html [hunk 1]
```diff
@@ -20,8 +20,8 @@ <h1>Eclipse Foundation Specification License - v1.0</h1>
     <li>All existing copyright notices, or if one does not exist, a notice
         (hypertext is preferred, but a textual representation is permitted)
         of the form: &quot;Copyright &copy; [$date-of-document]
-        &ldquo;Eclipse Foundation, Inc. &lt;&lt;url to this license&gt;&gt;
-        &quot;
+        &ldquo;Eclipse Foundation, Inc. <a href="https://www.eclipse.org/legal/efsl.php">
+        https://www.eclipse.org/legal/efsl.php</a>&quot;
     </li>
```
Comment (markt-asf) [inline api/src/main/javadoc/doc-files/speclicense.html:25]: The above changes are not correct. These was some discussion as the initial advice was incorrect. See EL, JSP and WebSocket for correct examples.

File: spec/src/main/asciidoc/license-efsl.adoc [hunk 2]
```diff
@@ -28,7 +28,7 @@ document, or portions thereof, that you use:
 * All existing copyright notices, or if one does not exist, a notice
   (hypertext is preferred, but a textual representation is permitted)
   of the form: "Copyright (c) [$date-of-document]
-  Eclipse Foundation, Inc. <<url to this license>>"
+  Eclipse Foundation, Inc. https://www.eclipse.org/legal/efsl.php[]"
```
Comment (markt-asf) [inline spec/src/main/asciidoc/license-efsl.adoc:32]: As above.

File: api/src/main/javadoc/doc-files/speclicense.html [hunk 3]
```diff
@@ -42,9 +42,10 @@ <h1>Eclipse Foundation Specification License - v1.0</h1>
 
 <p>The notice is:</p>
 
-<p>&quot;Copyright &copy; 2018 Eclipse Foundation. This software or
-    document includes material copied from or derived from [title and URI
-    of the Eclipse Foundation specification document].&quot;</p>
+<p>&quot;Copyright &copy; 2018, 2020 Eclipse Foundation. This software or
+    document includes material copied from or derived from Jakarta™ Servlet
+    <a href="https://jakarta.ee/specifications/servlet/5.0/">
+        https://jakarta.ee/specifications/servlet/5.0/</a>.&quot;</p>
```
Comment (markt-asf) [inline api/src/main/javadoc/doc-files/speclicense.html:49]: Jakarta is a registered trademark. Again, see EL, JSP and WebSocket for correct examples.

File: spec/src/main/asciidoc/license-efsl.adoc [hunk 4]
```diff
@@ -47,9 +47,9 @@ specification is expressly prohibited.
 
 The notice is:
 
-"Copyright (c) 2018 Eclipse Foundation. This software or
-document includes material copied from or derived from [title and URI
-of the Eclipse Foundation specification document]."
+"Copyright (c) 2018, 2020 Eclipse Foundation. This software or
+document includes material copied from or derived from Jakarta(TM) Servlet
+https://jakarta.ee/specifications/servlet/5.0/[]"
```
Comment (markt-asf) [inline spec/src/main/asciidoc/license-efsl.adoc:53]: As above

File: api/src/main/javadoc/doc-files/speclicense.html (hunk 1 above)
Comment (mcserra) [reply api/src/main/javadoc/doc-files/speclicense.html:24]: done

File: spec/src/main/asciidoc/license-efsl.adoc (hunk 2 above)
Comment (mcserra) [reply spec/src/main/asciidoc/license-efsl.adoc]: done

File: api/src/main/javadoc/doc-files/speclicense.html (hunk 3 above)
Comment (mcserra) [reply api/src/main/javadoc/doc-files/speclicense.html:48]: done

File: spec/src/main/asciidoc/license-efsl.adoc (hunk 4 above)
Comment (mcserra) [reply spec/src/main/asciidoc/license-efsl.adoc:53]: done

**Comment by arjantijms** [comment]
Closing as requested

**Pull request closed without merging**